test/eqDeep.js

```diff
@@ -0,0 +1,86 @@
+/*jshint typed: true */
```

Comment:
Nit: Please add a space between `*` and `j`.

Also, now's a good time to remove the line you added to __.jshintrc__.